From bb879b786eb8bf8e9714905c87465d9b1ed1ed94 Mon Sep 17 00:00:00 2001
From: zachary <zhangzg@marvell.com>
Date: Fri, 8 Apr 2016 00:48:14 +0800
Subject: [PATCH 1029/1240] fix: mbus: a3700: rename armadalp/arlp to a3700 for
 mbus

- rename armadalp-sb/nb to armada-3700-sb/nb, armadalp-mbus to
  armada-3700-mbus in armada-3700.dtsi file
- renmae _MBUS_ARLP_H_ to _MBUS_A3700_H_ in mbus_reg.h

Change-Id: I25c0c7641f225937992041798c1872da72143394
Signed-off-by: zachary <zhangzg@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/28435
Tested-by: Star_Automation <star@marvell.com>
Reviewed-by: Wilson Ding <dingwei@marvell.com>
Reviewed-by: Haim Boot <hayim@marvell.com>
---
 arch/arm/include/asm/arch-armada3700/mbus_reg.h | 8 ++++----
 drivers/misc/mvebu_mbus.c                       | 2 +-
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm/include/asm/arch-armada3700/mbus_reg.h b/arch/arm/include/asm/arch-armada3700/mbus_reg.h
index 03e29c2..5d77a8b 100644
--- a/arch/arm/include/asm/arch-armada3700/mbus_reg.h
+++ b/arch/arm/include/asm/arch-armada3700/mbus_reg.h
@@ -16,8 +16,8 @@
 * ***************************************************************************
 */
 
-#ifndef _MBUS_ARLP_H_
-#define _MBUS_ARLP_H_
+#ifndef _MBUS_A3700_H_
+#define _MBUS_A3700_H_
 
 #define MBUS_WIN_CTRL_REG(win_num)		(win_num * 0x10)
 #define MBUS_CR_WIN_ENABLE				0x1
@@ -40,10 +40,10 @@
 #define MBUS_RLR_REMAP_LOW_OFFS			0
 #define MBUS_RLR_REMAP_LOW_MASK			(0xffffff << MBUS_RLR_REMAP_LOW_OFFS)
 
-/* there is no remap_high reg in ArmadaLP, just define here to pass compilation */
+/* there is no remap_high reg in Armada3700, just define here to pass compilation */
 #define MBUS_WIN_REMAP_HIGH_REG(win_num)	(0xc + win_num * 0x10)
 
 #define MBUS_WIN_INTEREG_REG			(0xF0)
 
-#endif	/* _MBUS_ARLP_H_ */
+#endif	/* _MBUS_A3700_H_ */
 
diff --git a/drivers/misc/mvebu_mbus.c b/drivers/misc/mvebu_mbus.c
index 987d6bb..36f8adc 100644
--- a/drivers/misc/mvebu_mbus.c
+++ b/drivers/misc/mvebu_mbus.c
@@ -309,7 +309,7 @@ int init_mbus(void)
 */
 #if (defined(CONFIG_MVEBU_MBUS_SPL_ONLY) && defined(CONFIG_SPL_BUILD)) || !defined(CONFIG_MVEBU_MBUS_SPL_ONLY)
 #ifdef CONFIG_MVEBU_MBUS_SKIP_DRAM_WIN
-	/* for some Soc, like ArmadaLP, DRAM window has to be at the begining,
+	/* for some Soc, like Armada3700, DRAM window has to be at the beginning,
 	     and could not be configured by anyway (keep the default value),
 	     In this case, need to skip DRAM window configuration */
 	for (start_win_id = 0, win = memory_map; start_win_id < count; start_win_id++, win++)
-- 
1.9.1

